read_header.hpp

```diff
@@ -123,12 +149,22 @@ void read_cell_data(const std::string& cell_file, const Box& box, std::vector<fl
     int ny = box.yhi - box.ylo + 1;
     int nz = box.zhi - box.zlo + 1;
     int num_cells = nx * ny * nz;
+    
+
+    //print the location of the pointer
+    std::cout << "Current position: " << infile.tellg() << std::endl;
+
+    // Seek to the offset (skip first num_cells floats) // To read the velocity information
+    //infile.seekg(num_cells * sizeof(float), std::ios::beg);
+
+    //print the location of the pointer
+    std::cout << "Current position after seek: " << infile.tellg() << std::endl;
```

Comment:
These comments appear to be intended for general debugging. These should be added to the main repo.